pkg/dbal/Tests/DbalConnectionFactoryTest.php

```diff
@@ -28,4 +28,32 @@ public function testShouldCreateLazyContext()
         $this->assertAttributeEquals(null, 'connection', $context);
         $this->assertAttributeInternalType('callable', 'connectionFactory', $context);
     }
+
+    public function testShouldParseGenericDSN()
```

Comment:
move tests to DbalConnectionFactoryConfigTest

Reply:
Are you sure? I'm not testing the config, I'm trying to verify that the factory creates a valid configuration and no information is lost in the process.

Reply:
okay